appengine-java8/helloworld-springboot/pom.xml

```diff
@@ -1,5 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <!--
-~ Copyright (c) 2013 Google Inc.
+~ Copyright (c) 2020 Google LLC
```

Comment:
Doesn't need to be updated, but I guess the license checker asked you to do this.
@kurtisvg this should pass.

Reply:
Since I re-wrote most of this sample, would it be ok to update this?